app/main.py

```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(command: str) -> None:
+    files = command.strip("cp ").split()
+    if files[0] != files[1]:
+        with open(files[0], "r") as file_in, open(files[1], "w") as file_out:
```

Comment:
you are doing indexing twice so mb it worth to save file names in different variables instead of getting them by index each time